src/components/atoms/PriceCircle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface Props {
+  price: number;
+  className: string;
+}
+
+const StyledCircle = styled.div<Props>`
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: ${({ theme }) => theme.primaryColor};
+  font-size: ${({ price }) => {
+    const length: number = price.toString().length;
+    return Math.min(16, 18 - length);
+  }}px; // Epic hack for decreasing font-size for longer numbers
```

Comment:
lol :D nieźle

Reply:
Czasem sie we froncie przydaje matematyka xd jeden z niewielu przypadków